pkg/testing/deepcopy_test.go

```diff
@@ -45,34 +45,16 @@ var groups = map[schema.GroupVersion]runtime.SchemeBuilder{
 // TestRoundTripTypesWithoutProtobuf applies the round-trip test to all round-trippable Kinds
 // in the scheme.
 func TestRoundTripTypesWithoutProtobuf(t *testing.T) {
-	// TODO upstream this loop
+	scheme := runtime.NewScheme()
```

Comment:
do you mind structuring this as a call to another method so I can see the data being passed more clearly?